### ISSUE TITLE
Referring Wiki pages

### ISSUE THREAD

**Issue by lankavitharana** [ConsumerDataStandardsAustralia/standards-maintenance#417]
# Description
I have seen some spec pages refers to Wiki pages, which is not a good practice as wiki cannot be treated as a reliable source of truth, it is mentioned in Wikipedia itself too (https://en.wikipedia.org/wiki/Wikipedia:Wikipedia_is_not_a_reliable_source). It's kind of a contradictory statement, anyway since anyone can modify wiki pages, better to refer to reliable sources in the spec. 

# Area Affected
example area would be - https://consumerdatastandardsaustralia.github.io/standards/#tocSbankingscheduledpaymentinterval here it refers some "ISO 8601 Durations" to a wiki page - https://en.wikipedia.org/wiki/ISO_8601#Durations

# Change Proposed

**Comment by CDR-API-Stream** [comment]
Hi @lankavitharana that's a fair point. The DSB is not aware of an open / public standards reference for ISO8601. If you can provide one we would be happy to update the documentation. If the data standards point to a normative reference, we're mindful that it needs to be publicly accessible and not behind a paywall. You make a valid point regarding linking to a Wiki page however that's the only current reference to ISO8601 definitions available.

**Comment by lankavitharana** [comment]
@CDR-API-Stream yeah, seems ISO8601 needs to be purchased, however, given this is basically data protocol over the internet, how about using RFC standards? 
main site - https://www.ietf.org/standards/rfcs/
standard related to date and time - https://www.rfc-editor.org/rfc/rfc3339.html  (this is anyway profile of the ISO 8601)
all the RFC standards - https://www.rfc-editor.org/rfc-index.html
HTTP protocol in RFC for reference -  https://www.rfc-editor.org/rfc/rfc7230.html

**Comment by nils-work** [comment]
The only link in the Standards that refers to Wikipedia is for [ISO 8601 Durations](https://en.wikipedia.org/wiki/ISO_8601#Durations) for the reasons [above](https://github.com/ConsumerDataStandardsAustralia/standards-maintenance/issues/417#issuecomment-964777819).

There are four references to ISO in the Consumer Data Standards. 
The following updates to those references may be considered:

Current reference | For consideration
---- | ------
**ISO 3166 Alpha-3**<br>https://www.iso.org/iso-3166-country-codes.html | 1. Update to the location of the latest version: **[ISO 3166-1:2020](https://www.iso.org/standard/72482.html)**.<br>2. Link to the **[Documents in administration, commerce and industry](https://www.iso.org/ics/01.140.30/x/)** catalogue page where this standard can be found.<br>3. No change.
**ISO 8601 Durations**<br>https://en.wikipedia.org/wiki/ISO_8601#Durations | 1. Update to the location of the latest version: **[ISO 8601-1:2019](https://www.iso.org/standard/70907.html)**.<br>2. Link to the **[Documents in administration, commerce and industry](https://www.iso.org/ics/01.140.30/x/)** catalogue page where this standard can be found.<br>3. No change.
**ISO 9362**<br>https://www.iso.org/standard/60390.html | 1. Update to the location of the latest version: **[ISO 9362:2022](https://www.iso.org/standard/84108.html)**.<br>2. Link to the **[Finances. Banking. Monetary systems. Insurance](https://www.iso.org/ics/03.060/x/)** catalogue page where this standard can be found.<br>3. No change.
**ISO 17442**<br>https://www.iso.org/standard/59771.html | 1. Update to the location of the latest version: **[ISO 17442-1:2020](https://www.iso.org/standard/78829.html)**.<br>2. Link to the **[Finances. Banking. Monetary systems. Insurance](https://www.iso.org/ics/03.060/x/)** catalogue page where this standard can be found.<br>3. No change.
